### ISSUE TITLE
.git folder being distributed with package

### ISSUE THREAD

**Issue by felipeparaujo** [Cropster/ember-visual-test#32]
It looks like the `.git` is being distributed with `v0.3.1`, which makes `npm install` fail with the following error:

```
npm ERR! path /Users/felipearaujo/workspace/node_modules/ember-visual-test
npm ERR! code EISGIT
npm ERR! git /Users/felipearaujo/workspace/node_modules/ember-visual-test: Appears to be a git repo or submodule.
npm ERR! git     /Users/felipearaujo/workspace/node_modules/ember-visual-test
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/felipearaujo/.npm/_logs/2019-09-18T10_03_13_319Z-debug.log
```

**Comment by mydea** [comment]
I guess this is probably the same issue as here:
https://github.com/mydea/ember-date-components/issues/73

See: https://npm.community/t/npm-6-9-1-is-broken-due-to-git-folder-in-published-tarball/8454/5

Are you by change on npm 6.9.1? If so, please try updating, see if it works then (if not, please try removing the node_modules folder and re-installing).

**Comment by felipeparaujo** [comment]
I'm on v6.11.3. I tried deleting `node_modules`, didn't work either...

**Comment by mydea** [comment]
I just published v0.3.2, which should fix this! Not sure what exactly is going on, but I made sure to remove any .git folder in the npm package...!
